src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.cshtml

```diff
@@ -44,7 +44,12 @@
         </li>
     }
 </ul>
-
+@{
+    Model.Pager.FirstText = T["<i class='fa fa-xs fa-chevron-left'></i><i class='fa fa-xs fa-chevron-left'></i>"];
+    Model.Pager.PreviousText = T["<i class='fa fa-xs fa-chevron-left'></i>"];
```

Comment:
Why they are localized?!!

Reply:
Because the PagerShape requires an object to be passed on these. Not a string. So a StringLocalizer object.
When we try to pass an icon here it works but we should not do this.

Reply:
We should probably have Model.Pager.FirstIcon ...

Reply:
I'm looking at something about the CoerceHtmlString method in that PagerSahpes

Reply:
So basically T here is a IViewLocalizer which returns an IHtmlContent "I think". This is why it works from the view when doing this. Because it pass in the CoerceHtmlString method and returns a proper IHtmlContent result. Else if he would pass a string, it would return a StringHtmlContent(value.ToString())

Reply:
Like this : 

![image](https://user-images.githubusercontent.com/3228637/102519094-0fbcc680-4060-11eb-8850-54bde121c330.png)